### PR TITLE
More 1.4 release notes!

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: _includes/manuals/1.4/1.2_release_notes.md [hunk 1]
```diff
@@ -2,13 +2,45 @@
...
 
+#### Compute profiles
+
+Administrators are now able to define sets of default attributes for new VMs on each compute resource (of any type), and label it as an one-stop "compute profile" for users to create hosts with.
+
+This speeds up host creation and avoids mistakes with complex network and storage options being set by users each time.  Read more about configuring profiles in [Using Compute Profiles](/manuals/1.4/index.html#5.2.2UsingComputeProfiles).
+
+#### Image user data support
+
+Image-based compute resources (EC2, OpenStack Nova) now support supplying a template as user data to the VM instead of Foreman using SSH to configure the newly created host.  In particular, this allows hosts to be created when there is no direct network access from Foreman to the hosts.
```
Comment (GregSutcliffe) [inline _includes/manuals/1.4/1.2_release_notes.md]: Perhaps also asy that while no access needed from Foreman->client, access is instead required from client->Foreman?

File: _includes/manuals/1.4/1.2_release_notes.md [hunk 2]
```diff
@@ -2,13 +2,45 @@
...
+
+#### Web UI Kerberos support
+
+Foreman can now be configured under Apache HTTP with mod_auth_kerb to authenticate users using GSSAPI and Kerberos, allowing single sign on (SSO) in enterprise environments.  This has been tested with [FreeIPA project](http://www.freeipa.org) and more information on configuring it can be found on the [mod_auth_kerb wiki page](http://projects.theforeman.org/projects/foreman/wiki/Foreman_and_mod_auth_kerb).
+
+#### New extensible menu system for plugins
+
+Foreman 1.4 introduces a new plugin registration system, which allows plugins to extend the menu system, add permissions, roles and other enhancements to the Foreman installation when they're loaded.  Installed plugins are also now visible under *Administer > About*, `foreman-rake plugins` and in the API.
+
+A new "two pane" design has also been added to many index pages around the application, enabling quick creation and editing without waiting for the page to reload entirely.  Full page edits may still be opened in new tabs.
```
Comment (GregSutcliffe) [inline _includes/manuals/1.4/1.2_release_notes.md:25]: Without a separate heading it feels like this relates only to plugins.

**Comment by GregSutcliffe** [comment]
Generally :+1:, great work!

File: _includes/manuals/1.4/1.2_release_notes.md (hunk 2 above)
Comment (domcleal) [inline _includes/manuals/1.4/1.2_release_notes.md:25]: Fixed the heading, I want to keep to five points :)

File: _includes/manuals/1.4/1.2_release_notes.md (hunk 1 above)
Comment (domcleal) [inline _includes/manuals/1.4/1.2_release_notes.md]: Fixed, thanks.

**Pull request merged**